### PR TITLE
[TLM CI] Split tlm ci into two separate workflows

### PR DESCRIPTION
**Pull request by ulya-tkch**
+ remove extra tlm call from ci.ylm

### CODE REVIEW

**Comment by ulya-tkch** [comment]
> /test-tlm ✨ **Starting TLM tests...** ✨ If you want to run the TLM tests again (in case they failed or you have further updates to the TLM code), comment '/test-tlm' on this PR. [View full GitHub Actions run log](https://github.com/cleanlab/cleanlab-studio/actions/runs/10836161764)

FYI this will not pass correctly until this pr is in (will not have results reported)

File: .github/workflows/ci-test-tlm.yml [hunk 1]
```diff
@@ -45,9 +42,9 @@ jobs:
         run:  cleanlab login --key "${{ secrets.CLEANLAB_STUDIO_CI_API_KEY }}"
       - name: Run tests
         run: |
-          pytest tests/tlm/ --ignore=tests/tlm/test_properties.py --verbose
+          pytest --ignore=tests/tlm/test_properties.py --verbose
   propertytest:
-    name: "Property Test TLM: Python ${{ matrix.python }} on ${{ matrix.os }}"
+    name: "TLM Property Test TLM: Python ${{ matrix.python }} on ${{ matrix.os }}"
```
Comment (huiwengoh) [inline .github/workflows/ci-test-tlm.yml:47]:
```suggestion
    name: "TLM Property Test: Python ${{ matrix.python }} on ${{ matrix.os }}"
```

File: .github/workflows/ci-test-tlm.yml [hunk 2]
```diff
@@ -45,9 +42,9 @@ jobs:
         run:  cleanlab login --key "${{ secrets.CLEANLAB_STUDIO_CI_API_KEY }}"
       - name: Run tests
         run: |
-          pytest tests/tlm/ --ignore=tests/tlm/test_properties.py --verbose
+          pytest --ignore=tests/tlm/test_properties.py --verbose
```
Comment (huiwengoh) [inline .github/workflows/ci-test-tlm.yml:45]:
```suggestion
          pytest tests/tlm/ --ignore=tests/tlm/test_properties.py --verbose
```

**Review by huiwengoh — APPROVED** [review]
Just address nit comments

**Pull request merged**